### ISSUE TITLE
QC: Changing contexts in the file repository must reevaluate quality control results

### ISSUE THREAD

**Issue by urapadmin** [urapadmin/kiosk#1071]
when changing context - file relations in the edit dialog of file repository quality control rules should be recalculated for the involved contexts

**Comment by urapadmin** [comment]
the issue here is that detaching a file from a context (or even deleting the whole file) does not drop the locus_photo record but sets the image container in locus_photo to nothing. Quality Control still accepts that as an existing locus-photo record and so the qc rule, even though it runs again, has nothing to complain. This is a quixotic issue for quality control and can't be fixed in the current context.

**Comment by urapadmin** [comment]
### @test 0.12
- [ ] if a locus shows the warning that it should have an opening photo in the director's view and you add the locus as a context to a file in the file repository, on returning to the director's view the warning should be gone.
- [ ] Deleting or modifying a context or deleting a file has no effect on quality control in 0.12.

**Comment by luizaogs** [comment]
A question: the warning is gone but it still shows up as 0 photos on that previous day, though today's record for that locus shows up with a photo. I guess that's correct since the photo was added today and not previously? But wanted to check. 

I added a photo to LA test-006. If you go to 4 June 2021 you will see that the warning for the missing opening photo is gone but that there is a 0 in the photos column, and if you go to today in the director's view you'll see that the 1 for the photo shows up.
